WebHostLib/static/assets/player-settings.js

```diff
@@ -145,8 +145,13 @@ const buildOptionsTable = (settings, romOpts = false) => {
         range.setAttribute('min', settings[setting].min);
         range.setAttribute('max', settings[setting].max);
         range.value = currentSettings[gameName][setting];
+        const settingMax = settings[setting].max;
+        const specialMax = settings[setting].specialMax;
         range.addEventListener('change', (event) => {
-          document.getElementById(`${setting}-value`).innerText = event.target.value;
+          const v = event.target.value;
+          // if at max and have specialMax, use that, else use value
+          const text = ((v == settingMax) && specialMax) || v;
+          document.getElementById(`${setting}-value`).innerText = text;
```

Comment:
This will cause the layout of the page to change suddenly and without warning. This behavior is undesirable.

Reply:
First, I consider this discovery to be a bug report and I will be addressing it.

Second, sudden layout changes are undesirable and should be avoided.

Third, and in my opinion most importantly, those range sliders are intended for use with numbers only. Words were never supposed to be used in place of those numbers.

Reply:
Do you have any better ideas for how to have an option that can be from a large range of numbers and one or a few other non-number values?

Multiple people have said they would like this, and Berserker suggested that I implement this.

Reply:
As indicated in my final comment on the PR, I will be working on a new `range` control to accommodate for this.